.github/setup-node/action.yml

```diff
@@ -42,6 +42,19 @@ runs:
               name: npm-logs
```

Comment:
Need to remove this task.

Reply:
Ref: https://github.com/WordPress/gutenberg/pull/61289#issuecomment-2093688244

Reply:
@DaniGuardiola Could you have another look? I've merged `trunk` into this PR and this should now be addressed.